dev/tasks/exec/tasks.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const gulp = require( 'gulp' );
+const gutil = require( 'gulp-util' );
+const minimist = require( 'minimist' );
+
+const exec = require( './tasks/exec' );
+const log = require( '../../lib/log' );
+
+module.exports = ( config ) => {
+	const ckeditor5Path = process.cwd();
+	const packageJSON = require( '../../../package.json' );
+
+	// Configure logging.
+	log.configure(
```

Comment:
Harcoding will be the simplest option. We don't have a use case for configuring that logger other than tests. That's why I asked whether they pass.